demos/demo-react-ts/src/components/AvailabilityDropdown.tsx

```diff
@@ -0,0 +1,175 @@
+import { useMemo, useCallback, useState, ReactElement } from "react";
+
+import {
+  AvailabilityButton,
+  AvailabilityTooltip,
+  AvailabilityToggleButton,
+  AvailabilityOption,
+  Divider,
+  AvailabilityButtonContent,
+  AvailabilityToggleButtonContent,
+  TriggerIncomingCallTooltip,
+  Input,
+  DialIncomingCallButton,
+} from "./Components";
+import {
+  AvailabilityCheckmarkSvg,
+  AvailableStatusDotSvg,
+  CaretDownSvg,
+  StartCallSvg,
+  UnavailableStatusDotSvg,
+} from "./Icons";
+import { Availability } from "../types/ScreenTypes";
+import {
+  validateKeypadInput,
+  validatePhoneNumber,
+} from "../utils/phoneNumberUtils";
+
+function getAvailabilityString(availability: Availability): string {
+  return availability === "AVAILABLE" ? "Available" : "Unavailable";
+}
+
+function getAvailabilityStatusDot(availability: Availability): ReactElement {
+  return availability === "AVAILABLE"
+    ? AvailableStatusDotSvg
+    : UnavailableStatusDotSvg;
+}
+
+function AvailabilityDropdown({
+  availability,
+  setAvailability,
+  triggerIncomingCall,
+  incomingNumber,
+  setIncomingNumber,
+}: {
+  availability: Availability;
+  setAvailability: Function;
+  triggerIncomingCall: Function;
+  incomingNumber: string;
+  setIncomingNumber: Function;
+}) {
+  const [toggleAvailabilityOptions, setToggleAvailabilityOptions] =
+    useState<boolean>(false);
+  const [toggleTriggerIncomingCall, setToggleTriggerIncomingCall] =
+    useState<boolean>(false);
+  const handleAvailability = useCallback(
+    (availability: string) => {
+      setAvailability(availability);
+    },
+    [setAvailability]
+  );
+
+  const IncomingCallInput = useMemo(() => {
+    const isIncomingNumberValid =
+      validateKeypadInput(incomingNumber) &&
+      validatePhoneNumber(incomingNumber);
+    const handleInput = (event: React.ChangeEvent<HTMLInputElement>) => {
+      setIncomingNumber(event.target.value);
+    };
+    return (
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "column",
+          rowGap: 10,
+          alignItems: "center",
+        }}
+      >
+        <Input onChange={handleInput} value={incomingNumber} />
+        <DialIncomingCallButton
+          use="primary"
+          disabled={!isIncomingNumberValid}
+          onClick={() => triggerIncomingCall(incomingNumber)}
+          aria-label="trigger-incoming-call"
+        >
+          {StartCallSvg}
+        </DialIncomingCallButton>
+      </div>
+    );
+  }, [triggerIncomingCall, incomingNumber, setIncomingNumber]);
+
+  const AvailabilityOptions = useMemo(() => {
+    return (
+      <div>
```

Comment:
The CSS here looks a little bit different from the mock up:
- The "Available" font should be the same "Log Out"
- Line separating availability statuses and button needs to be slightly more transparent and cover the full width of div
- Trigger incoming call button background
Code
<img width="404" alt="image" src="https://github.com/HubSpot/calling-extensions-sdk/assets/23175119/69f1bac1-c0f5-4676-aa78-fb6bab71a48e">

Mock Up
<img width="421" alt="image" src="https://github.com/HubSpot/calling-extensions-sdk/assets/23175119/b1af4bc4-8137-4d02-b56c-be628395c88f">

Reply:
Fixed